TicTacToeTests/BoardTest.cs

```diff
@@ -16,53 +16,48 @@ public class BoardTest
         public void aNewBoardIsEmpty()
         {
             var subject = new Board();
-            Assert.True(subject.isEmpty());
+            Assert.True(subject.isBoardEmpty());
         }
 
-
         [Fact]
         public void aBoardCanMarkaSpace()
         {
             var subject = new Board();
-            Assert.True(subject.placeMarker(5, P2_MARKER));
+            subject.placeMarker(5, P2_MARKER);
+            Assert.Equal(P2_MARKER, subject.markerAtLocation(5));
         }
 
-
         [Fact]
         public void aNewBoardContainsInstancesOfSpaces()
```

Comment:
You want to make sure you're testing behavior and not implementation, otherwise refactoring will be super painful. That a board is made of spaces is an implementation detail and not crucial to the correct behavior of the TTT game. You can use these kinds of tests as "stepping stones" if it's helpful in TDD but go back and delete them eventually.